### ISSUE TITLE
support hex color

### ISSUE THREAD

**Issue by kzfile** [weatherlayers/cpt2js#3]
Hi, I get NaN if I set color table in hex string like:
```javascript
const { parsePalette } = require('cpt2js');

const palette = `
0   #5d96a5
1   #91d54d
`;

const paletteScale = parsePalette(palette);

paletteScale(1).rbg(); // [ NaN, NaN, NaN ]
```
I notice the color test [here](https://github.com/weatherlayers/cpt2js/blob/13a9d402b69c37d0c7d9ce4eee358d0e99674dd1/src/parse.ts#L76) ignore hex string color, does it meet the expectations?

Thank You.

**Comment by kzfile** [comment]
@zakjan is this a feature or a bug?

**Comment by zakjan** [comment]
Hi, yes this was a bug indeed. Thanks for reporting.

Fixed in https://github.com/weatherlayers/cpt2js/commit/2da0c24516367ca469b293805fbe1b7bcfb5aaed, released as 1.5.2.
